stdlib/src/math.rs

```diff
@@ -211,7 +211,9 @@ mod math {
     #[pyfunction]
     fn sqrt(value: ArgIntoFloat, vm: &VirtualMachine) -> PyResult<f64> {
         let value = *value;
-        if value.is_nan() { return Ok(value); }   //return nan if value is nan
+        if value.is_nan() {
+            return Ok(value);
+        } //return nan if value is nan
```

Comment:
Don't think this really requires a comment here, can just remove it:

```suggestion
        if value.is_nan() {
            return Ok(value);
        }
```